### ISSUE TITLE
[REQUEST] Make the Alarm (color) feature blinkable

### ISSUE THREAD

**Issue by Bhaskar-S-Jois** [agronick/aa-torque#124]
**Is your feature request related to a problem?**- No, the feature is a cool one.

**Please describe.**
Make the Alarm (color) feature blinkable with option to customise the blink frequency 

**Describe the solution you'd like**
Taking example for an RPM redline, fuel warning or high temperature warning, it would be more impactful and real if the colors could toggle on and off with custom frequency to indicate the criticality. 
For example, RPM reading to go green between 0 and 2000, yellow between 2000 and 3000, orange between 3000 and 4000, red between 4000 and 5000, blinking red above 5000. This would actually emulate the real RPM redline scenario.

**Comment by bgeneto** [comment]
Agreed. We can barely see the alarm arc below the display. It is also too thin...
